### ISSUE TITLE
HTML exporting worng charset

### ISSUE THREAD

**Issue by alansenairj** [trapias/TrelloExport#83]
Output HTML export file function is writing down all  ISO 8559-1  chars instead UTF-8. 
I'm using Ubunt 21.04, both Chrome and Firefox browsers. Page loads fine but export result file is poluted by substituitions.

**Comment by trapias** [comment]
@alansenairj can you show me an example?

**Comment by alansenairj** [comment]
Yes. The problem is markdown conversion when someone put code on Trello card.  Export Results: 

![image](https://user-images.githubusercontent.com/20565821/123264281-40c6f980-d4d0-11eb-9203-466952913a8a.png)

Original card: 

![image](https://user-images.githubusercontent.com/20565821/123264457-710e9800-d4d0-11eb-9c84-17df6d7a2fe2.png)


Markdown text: 

# Comandos Básicos do GIT

## Iniciar um repositório

Esse comando irá transformar um diretório simples em um repositório git.

``` $ git init ```

## Git config

É possível alterar algumas configurações do git como o editor, paginador, usuário, email, seguem alguns exemplos:

Para alterar as configurações para todos os repositórios utilizar a flag ```--global``` após a opção config

Mudando o paginador do comando "**less**" para o comando "**cat**"
``` $ git config core-pager cat```

Mudando as informações de usuário

**Comment by alansenairj** [comment]
II'm using python to convert markdown export instead HTML.

**Comment by trapias** [comment]
@alansenairj will try to understand what I can do; if you open the resulting HTML in the browser isn't it properly shown?
Eventually tell us what're doing with python, maybe the trick can help somebody else :-)
